srtcore/api.cpp

```diff
@@ -254,18 +254,24 @@ int CUDTUnited::startup()
 
    PacketFilter::globalInit();
 
-   //init CTimer::EventLock
-
    if (m_bGCStatus)
-      return true;
+      return 1;
 
    m_bClosing = false;
 
    setupMutex(m_GCStopLock, "GCStop");
    setupCond(m_GCStopCond, "GCStop");
    {
        ThreadName tn("SRT:GC");
```

Comment:
As we already have our own thread wrappers, it's worth to drag this ThreadName functionality into the wrapper. Although we can think about this later when gluing with thread logging wrappers.

Reply:
For now `CThread` is to be mapped directly to C++11 `std::thread`, therefore this `ThreadName` stays outside of the class. Can be changed in the future if provides benefits.

Reply:
Well, depends. If you already must have a dedicated function for assigning a thread, you can hide the thread-name functionality inside it.

Reply:
The thre-name functionality is hidden into the StartThread(..) function now.